src/device.test.js

```diff
@@ -0,0 +1,26 @@
+const test = require('ava');
+const device = require('./device');
+
+test('returns the correct device info', t => {
+  const actual = device({
+    tuner_count: 12,
+    device_uuid: '000-000-000',
+    antennas_url: 'https://antennas.test'
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/jfarseneau/antennas/pullRequest?prid=8719173)
```suggestion
    antennas_url: "https://antennas.test"
```